charts/kuberpult/values.yaml.tpl

```diff
@@ -199,22 +199,48 @@ auth:
     cloudInstance: "https://login.microsoftonline.com/"
     clientId: ""
     tenantId: ""
+  dexAuth:
+    enabled: false
+    # Indicates if dex is to be installed. If you want to use your own Dex instance do not enable this flag.
+    installDex: false
+    # Defines the rbac policy when using Dex.
+    # The permissions are added using the following format (<ROLE>, <ACTION>, <ENVIRONMENT_GROUP>:<ENVIRONMENT>, <APPLICATION>, allow).
+    #
+    # Available actions are: CreateLock, DeleteLock, CreateRelease, DeployRelease, CreateUndeploy, DeployUndeploy, CreateEnvironment, CreateEnvironmentApplication and DeployReleaseTrain.
+    # The actions CreateUndeploy, DeployUndeploy and CreateEnvironmentApplication are environment independent meaning that the environment specified on the permission
+    # needs to follow the following format <ENVIRONMENT_GROUP>:*, otherwise an error will be thrown.
+    #
+    # Example permission: Developer, CreateLock, development:development, *, allow
+    # If no group is configured for an environment, the environment group name is the same as the environment name, here "development".
+    # The policy will be available on the kuberpult-rbac config map.
+    policy_csv: ""
+    clientId: ""
+    clientSecret: ""
+    baseURL: ""
+    # List of scopes to validate the token. Please add them as comma separated values.
+    scopes: ""
+    # List of environment variables to be added to the dex service pod.
+    # Example, if you want your DEX service to have access to to the OAUTH_CLIENT_ID, you can specify
+    # it the following way: 
+    # 
+    #  - name: OAUTH_CLIENT_ID
+    #    valueFrom:
+    #      secretKeyRef:
+    #      name: kuberpult-oauth-client-id
+    #        key: kuberpult-oauth-client-id
+    envVars: []
```

Comment:
This config must be under `dex`, right? Otherwise the dex chart won't pick it up.

Reply:
Yes, you are right. Made the necessary changes and re tested everything locally on my team dummy cluster.